src/main/scala/com/databricks/labs/tika/TikaExtractor.scala

```diff
@@ -45,6 +46,7 @@ object TikaExtractor {
       val contentType = retrieveContentType(parser, metadata, stream, filename)
 
       // Extract content using the appropriate parsing
+      IOUtils.setByteArrayMaxOverride(byteArrayMaxOverride)
```

Comment:
As static method, I guess that may affect concurrent requests. Any idea how to make this change local?

Reply:
Good point. We may be able to do a tear down once complete to set back to the original setting. Will look into this

Reply:
Seems like `IOUtils.setByteArrayMaxOverride` is the only valid method for interacting with this. I am writing a config manager which can reset the static method to its original value (`-1`) during extraction. Since this will be passed to each worker at runtime, it shouldn't affect downstream requests.